shell/callbacks.c

```diff
@@ -44,13 +44,23 @@ void cb_save_graphic()
     gchar *filename;
```

Comment:
The `cb_save_graphic()` function can be removed as well, nothing calls it any longer.